From 35e6f623097c0221e2b3f14b0831f7b2bece1481 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Fri, 25 Mar 2016 18:01:06 +0800
Subject: [PATCH] Revert "iommu: rk-iovmm: change compatible name to a unified
 name"

This reverts commit 3a1bdfa3a4bd6e3bb0a1328ef50a4d3278560222.

This patch broken old rkfb driver, so revert it.

Change-Id: I7ccd93d8ff2086cca9c1b31932278435cbffc59f
---
 include/linux/rockchip-iovmm.h | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/include/linux/rockchip-iovmm.h b/include/linux/rockchip-iovmm.h
index 8f2dd0747484..ab7f8bb61f0f 100644
--- a/include/linux/rockchip-iovmm.h
+++ b/include/linux/rockchip-iovmm.h
@@ -11,15 +11,15 @@
 #include <linux/atomic.h>
 #include <linux/spinlock.h>
 
-#define IEP_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
-#define VIP_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
-#define ISP_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
-#define VOPB_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
-#define VOPL_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
-#define VOP_IOMMU_COMPATIBLE_NAME	"rockchip,iommu"
-#define HEVC_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
-#define VPU_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
-#define VDEC_IOMMU_COMPATIBLE_NAME "rockchip,iommu"
+#define IEP_IOMMU_COMPATIBLE_NAME "rockchip,iep_mmu"
+#define VIP_IOMMU_COMPATIBLE_NAME "rockchip,vip_mmu"
+#define ISP_IOMMU_COMPATIBLE_NAME "rockchip,isp_mmu"
+#define VOPB_IOMMU_COMPATIBLE_NAME "rockchip,vopb_mmu"
+#define VOPL_IOMMU_COMPATIBLE_NAME "rockchip,vopl_mmu"
+#define VOP_IOMMU_COMPATIBLE_NAME	"rockchip,vop_mmu"
+#define HEVC_IOMMU_COMPATIBLE_NAME "rockchip,hevc_mmu"
+#define VPU_IOMMU_COMPATIBLE_NAME "rockchip,vpu_mmu"
+#define VDEC_IOMMU_COMPATIBLE_NAME "rockchip,vdec_mmu"
 
 enum rk_iommu_inttype {
 	IOMMU_PAGEFAULT,
-- 
2.35.3

